src/modules/map/components/LocationPopup.vue

```diff
@@ -0,0 +1,228 @@
+                </div>
+                <div>
+                    <a :href="$t('contextpopup_lv03_url')" target="_blank">CH1903 / LV03</a>
+                </div>
+                <div data-cy="location-popup-coordinates-lv03">
+                    {{ clickCoordinatesLV03 }}
+                </div>
+                <div>
+                    <a :href="$t('contextpopup_wgs84_url')" target="_blank">WGS 84 (lat/lon)</a>
+                </div>
+                <div>
+                    <div data-cy="location-popup-coordinates-plain-wgs84">
+                        {{ clickCoordinatesPlainWGS84 }}
+                    </div>
+                    <div data-cy="location-popup-coordinates-wgs84">
+                        {{ clickCoordinatesWGS84 }}
+                    </div>
+                </div>
+                <div>
+                    <a :href="$t('contextpopup_utm_url')" target="_blank">UTM</a>
+                </div>
+                <div data-cy="location-popup-coordinates-utm">
+                    {{ clickCoordinatesUTM }}
+                </div>
+                <div>{{ 'MGRS' }}</div>
+                <div data-cy="location-popup-coordinates-mgrs">
+                    {{ clickCoordinatesMGRS }}
+                </div>
+                <div>
+                    <a :href="$t('contextpopup_what3words_url')" target="_blank">what3words</a>
+                </div>
+                <div>
+                    <span v-show="clickWhat3Words" data-cy="location-popup-w3w">{{
+                        clickWhat3Words
+                    }}</span>
+                </div>
+                <div>{{ $t('elevation') }}</div>
+                <div>
+                    <span v-if="height" data-cy="location-popup-height">
+                        {{ height.heightInMeter }} m / {{ height.heightInFeet }} ft
+                    </span>
+                </div>
+                <div></div>
+                <div data-cy="location-popup-link-bowl-crosshair">
+                    <a :href="shareLinkUrl" target="_blank">
+                        {{ $t('link_bowl_crosshair') }}
+                    </a>
+                </div>
+                <div class="qrcode-container">
+                    <img v-if="qrCodeImageSrc" :src="qrCodeImageSrc" />
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import proj4 from 'proj4'
+import { mapState, mapActions } from 'vuex'
+import { ClickType } from '@/modules/map/store/map.store'
+import Overlay from 'ol/Overlay'
+import OverlayPositioning from 'ol/OverlayPositioning'
+import { printHumanReadableCoordinates, CoordinateSystems } from '@/utils/coordinateUtils'
+import ButtonWithIcon from '@/utils/ButtonWithIcon'
+import { registerWhat3WordsLocation } from '@/api/what3words.api'
+import { requestHeight } from '@/api/height.api'
+import { generateQrCode } from '@/api/qrcode.api'
+import { round } from '@/utils/numberUtils'
+
+export default {
```

Comment:
Could you add a little comment (JSDoc) on what this component does (a quick description so that we don't have to read the whole component to know what it's doing)?